Dockerfile

```diff
@@ -14,8 +13,9 @@ RUN useradd -m csc3050 && \
 ENV LANG=en_US.utf8 \
 	LANGUAGE=en_US.UTF-8 \
 	LC_ALL=en_US.UTF-8
-RUN sudo -u csc3050 env HTTP_PROXY="$HTTP_PROXY" HTTPS_PROXY="$HTTPS_PROXY"  pikaur -S --noconfirm --color=always code-server
-RUN pacman -Sc --noconfirm && chsh -s /usr/bin/fish csc3050
+RUN sudo -u csc3050 pikaur -S --noconfirm --color=always code-server && \
```

Comment:
Would you please keep HTTP_PROXY envs so that  the local settings can work?
see https://docs.docker.com/network/proxy/
```
{
 "proxies":
 {
   "default":
   {
     "httpProxy": "http://127.0.0.1:3001",
     "httpsProxy": "http://127.0.0.1:3001",
     "noProxy": "*.test.example.com,.example2.com"
   }
 }
}
```
The settings will set up these envs, so remove the env is actually discarding the global envs of the host which is not good.

Or, maybe just switch user and get rid of `sudo`